packages/block-library/src/image/edit.native.js

```diff
@@ -309,7 +343,7 @@ export class ImageEdit extends React.Component {
 							hideCancelButton
 							icon={ 'editor-expand' }
 							label={ __( 'Size' ) }
-							value={ sizeOptionLabels[ sizeSlug || DEFAULT_SIZE_SLUG ] }
+							value={ sizeSlug }
```

Comment:
Not using the `sizeOptionLabels` here is causing the active size slug to display lower case (i.e. "large") in the UI. I think we want to still use the labels here.

![Screen Shot 2019-11-20 at 11 43 38 AM](https://user-images.githubusercontent.com/4656348/69258687-00342b00-0b8b-11ea-95ad-3f8bffce4fd4.png)

Reply:
You are completely right. I'll revert this change.